Resources/Prototypes/Entities/Objects/Tools/emag.yml

```
@@ -3,12 +3,54 @@
   - type: Emag
     blacklist: # DeltaV - Split Emag into 2 items, this one cannot emag doors
       components:
       - Airlock
+  - type: ExtendDescription
+    descriptionList:
+      - description: "emag-extenddescription-command"
+        fontSize: 12
+        color: "#1155ff"
+        requireDetailRange: false
+        requirements:
+        - !type:CharacterDepartmentRequirement
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 10 but found 8

(indentation)